shaptools/netweaver.py

```diff
@@ -112,16 +112,27 @@ def _is_ascs_installed(processes):
         Check if ASCS instance is installed
         """
         msg_server = shell.find_pattern(r'msg_server, MessageServer,.*', processes.output)
-        enserver = shell.find_pattern(r'enserver, EnqueueServer,', processes.output)
+        enserver = shell.find_pattern(r'enserver, EnqueueServer,.*', processes.output)
         return bool(msg_server and enserver)
 
     @staticmethod
     def _is_ers_installed(processes):
         """
         Check if ERS instance is installed
         """
-        msg_server = shell.find_pattern(r'enrepserver, EnqueueReplicator.*', processes.output)
-        return bool(msg_server)
+        enrepserver = shell.find_pattern(r'enrepserver, EnqueueReplicator,.*', processes.output)
+        return bool(enrepserver)
+
+    @staticmethod
+    def _is_app_server_installed(processes):
+        """
+        Check if an application server (PAS or AAS) instance is installed
+        """
+        disp = shell.find_pattern(r'disp\+work, Dispatcher,.*', processes.output)
+        igswd = shell.find_pattern(r'igswd_mt, IGS Watchdog,.*', processes.output)
+        gwrd = shell.find_pattern(r'gwrd, Gateway,.*', processes.output)
```

Comment:
this acroynyms aren't really so easy to understand unless you are in the process.. Can we add perhaps comments in the function comments? :thinking:

Reply:
The truth is that I don't even know what these processes mean. What I know is that all of them must be installed when the primary and additional services are installed.
If you follow a bit where this method is used, it just basically check the content of an output of a command (GetProcessList). Each Netweaver instance has different processes (as I said I don't know exactly what each process is doing, but actually I don't even need to know).